<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wStart</class>
 <widget class="QMainWindow" name="wStart">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>641</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Loading...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressources.qrc">
    <normaloff>:/rec/data/icons/iconOMSI-Tools.svg</normaloff>:/rec/data/icons/iconOMSI-Tools.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QFrame" name="fraUpdate">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblUpdateIcon">
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../ressources.qrc">:/rec/data/icons/iconDownload_blue.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblUpdate">
         <property name="text">
          <string notr="true">update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblUpdateVersion">
         <property name="font">
          <font>
           <family>Consolas</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #008AFF</string>
         </property>
         <property name="text">
          <string notr="true">version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnView">
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="gbxContentDevelopment">
          <property name="toolTip">
           <string/>
          </property>
          <property name="title">
           <string>Content development</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="btnFonts">
             <property name="toolTip">
              <string/>
             </property>
             <property name="whatsThis">
              <string/>
             </property>
             <property name="text">
              <string>Font creation</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconChar.svg</normaloff>:/rec/data/icons/iconChar.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnVehicles">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Vehicles</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSceneryobjects">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Sceneryobjects</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMoneyTicketpacks">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Money / ticket packs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnScripts">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Scripts</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMaps">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Maps</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTextures">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Textures</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSplines">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Splines</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbxTroubleshooting">
          <property name="toolTip">
           <string/>
          </property>
          <property name="title">
           <string>Troubleshooting</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="btnVerifyMap">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Verify map</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconSearch.svg</normaloff>:/rec/data/icons/iconSearch.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnContentSearch">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Content search</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconSearch.svg</normaloff>:/rec/data/icons/iconSearch.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="gbxMiscellaneous">
          <property name="toolTip">
           <string/>
          </property>
          <property name="title">
           <string>Miscellaneous</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="btnBackup">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="statusTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="text">
              <string>Backup</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconRedo.svg</normaloff>:/rec/data/icons/iconRedo.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="toolTip">
              <string/>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbxFinishing">
          <property name="toolTip">
           <string/>
          </property>
          <property name="title">
           <string>Finishing</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="btnSaveMod">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Save modifiaction for release...</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconSave.svg</normaloff>:/rec/data/icons/iconSave.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnWhereShouldIReleaseMyModification">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>This module is currently not available.</string>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="text">
              <string>Release modification</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconWorld.svg</normaloff>:/rec/data/icons/iconWorld.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSimulateCrash">
          <property name="text">
           <string>Simulate chrash</string>
          </property>
          <property name="icon">
           <iconset resource="../../ressources.qrc">
            <normaloff>:/rec/data/icons/iconCrash.svg</normaloff>:/rec/data/icons/iconCrash.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbxFbh">
          <property name="title">
           <string notr="true">Filebase-Helper</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QPushButton" name="btnOpenFbh">
             <property name="text">
              <string>Open</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconFbh.png</normaloff>:/rec/data/icons/iconFbh.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCreateShortcut">
             <property name="text">
              <string>Create shortcut</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconLink.svg</normaloff>:/rec/data/icons/iconLink.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbxDevTools">
          <property name="toolTip">
           <string/>
          </property>
          <property name="title">
           <string notr="true">DevTools</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QPushButton" name="btnDevTools">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string notr="true">DevTools</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconToolbox.svg</normaloff>:/rec/data/icons/iconToolbox.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDBPanel">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string notr="true">Content search DBPanel</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconDatabase.svg</normaloff>:/rec/data/icons/iconDatabase.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStyleTest">
             <property name="text">
              <string notr="true">StyleTest</string>
             </property>
             <property name="icon">
              <iconset resource="../../ressources.qrc">
               <normaloff>:/rec/data/icons/iconBroom.svg</normaloff>:/rec/data/icons/iconBroom.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Specials</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionSendFeedback"/>
    <addaction name="actionChangelog"/>
    <addaction name="actionAbout"/>
    <addaction name="actionSourceCodeOnGitHub"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconQuit.svg</normaloff>:/rec/data/icons/iconQuit.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+W</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconSettings.svg</normaloff>:/rec/data/icons/iconSettings.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconInfo.svg</normaloff>:/rec/data/icons/iconInfo.svg</iconset>
   </property>
   <property name="text">
    <string notr="true">About</string>
   </property>
   <property name="iconText">
    <string notr="true">About</string>
   </property>
   <property name="toolTip">
    <string notr="true">About</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionSendFeedback">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconBug.svg</normaloff>:/rec/data/icons/iconBug.svg</iconset>
   </property>
   <property name="text">
    <string>Send feedback</string>
   </property>
   <property name="shortcut">
    <string notr="true">F1</string>
   </property>
  </action>
  <action name="actionChangelog">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconSwitch.svg</normaloff>:/rec/data/icons/iconSwitch.svg</iconset>
   </property>
   <property name="text">
    <string>Changelog</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconBook.svg</normaloff>:/rec/data/icons/iconBook.svg</iconset>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionSourceCodeOnGitHub">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconGitHub.svg</normaloff>:/rec/data/icons/iconGitHub.svg</iconset>
   </property>
   <property name="text">
    <string>Source code on GitHub</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
